cred-manager/package.json

```diff
@@ -9,6 +9,8 @@
     "prebuild": "rimraf dist",
     "build": "nest build",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "sqllite:generate": "npx prisma generate --schema prisma-lite/schema.prisma",
+    "sqllite:push": "npx prisma db push --schema prisma-lite/schema.prisma",
```

Comment:
From what I understood, we are not using the prisma driver with the sqlite db right? And we are using the better-sqlite for that, if that's the case, we can delete these scripts